build-tools/automation/azure-pipelines.yaml

```diff
@@ -28,6 +28,7 @@ variables:
   1ESWindowsImage: AzurePipelinesWindows2022compliant
   1ESMacPool: Azure Pipelines
   1ESMacImage: internal-macos-11
+  Codeql.Enabled: true
```

Comment:
We may want to limit the variable scope since the pipeline has four jobs that include a product build and would potentially run these steps.

Could we set this at a job scope below instead, maybe on the Windows .NET Build here https://github.com/xamarin/java.interop/blob/main/build-tools/automation/azure-pipelines.yaml#L88?

Reply:
Good idea, thanks!